chapters/unitexpressions.tex

```diff
@@ -104,3 +104,34 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 
 The unit expression \lstinline!"T"! means tesla, but note that the letter \lstinline!T! is also the symbol for the prefix tera which has a multiplier value of 10\textsuperscript{12}.
 \end{example}
+
+
+\section{Unitful Literals}
+
+The \lstinline[language=grammar]!numeric! in the grammar allows a unit to be attached to a numeric value.
+In this case, the unit is given as a \lstinline[language=grammar]!Q-IDENT! rather than a string, since the use of a string could be ambiguity interpreted as \lstinline[language=grammar]!description-string! in some contexts.
```

Comment:
```suggestion
In this case, the unit is given as a \lstinline[language=grammar]!Q-IDENT! rather than a string, since the use of a string could be ambiguously interpreted as \lstinline[language=grammar]!description-string! in some contexts.
```
To me "ambiguity interpreted" looks odd, and I prefer "ambiguously interpreted"; (possibly with "be" in the middle)

https://books.google.com/ngrams/graph?content=ambiguity+interpreted%2C+ambiguously+interpreted&year_start=1800&year_end=2022&corpus=en&smoothing=3